v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/IStreamingCollectorPlugin.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SnapPluginLib
+{
+    public interface IStreamingCollectorPlugin : IPlugin
```

Comment:
streaming also need to define StreamingCollect only

Reply:
It implements ->https://github.com/solarwinds/snap-plugin-lib/blob/feature/streaming-collector-binding-support/v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/IStreamingCollectorPlugin.cs#L5